File: Part1_강의자료/Part1_쪽지시험/문제별코드/ProblemA_방번호/1475.py
N = int(input())
cnt = [0]*10

while N > 0:
    cnt[N%10] += 1
    N //= 10

ans = (cnt[6] + cnt[9] + 1) // 2
for i in range(9):
    if i != 6:
        ans = max(ans, cnt[i])
    
print(ans)

File: Part1_강의자료/Part1_쪽지시험/문제별코드/ProblemA_방번호/1475.cpp
#include <stdio.h>
#include <algorithm>
using namespace std;

int main()
{
    int N;
    int cnt[10] = {0, };

    scanf("%d", &N);
    while (N > 0)
    {
        cnt[N % 10]++;
        N /= 10;
    }

    int ans = (cnt[6] + cnt[9] + 1) / 2;
    for (int i = 0; i < 9; i++)
        if (i != 6)
            ans = max(ans, cnt[i]);
    
    printf("%d\n", ans);
    return 0;
}
